### DISCUSSION TITLE
Demand for blockchain developers?

### DISCUSSION THREAD

**Discussion by Anshpatwal** [Cyfrin/foundry-full-course-cu#2400] [General]
hey , I am just starting this course but i wanna know the current market for blockchain developers, if there a decent demand in market , also would love guidance for blockchain, please do answer if you are in the industry and have experience, need support also Good luck to others who are starting this .

**Comment by EngrPips** [comment]
Hello @Anshpatwal, I wish you the best of luck in your journey, I would have loved to answer this but I don't know enough about the market but I trust someone who knows will answer this soon.

**Reply by Anshpatwal** [reply]
Thanks Man , I wish you the best too.

**Reply by EngrPips** [reply]
Anytime.
